### PR TITLE
Updated terms for minimum/security stock status labels

### LINKED ISSUE #5072: Rename Low Stock / Critical Stock variables for better understanding
We have two confusing concepts: risk of stock out and risk of stock expiry.  I propose that we refer to any stock that is below the "minimum stock" as "Low Stock".  Anything below the "Security Stock" should be labeled "Critical Stock".

These names better convey the urgency of the issue and are also clearer than the academic concept of minimum/security.

@jmcameron as my english fact-checker, what do you think?

### PR DESCRIPTION
**Pull request by jmcameron**
The goal is to make these status labels easier to understand for typical users.
"Minimum Stock Reached" -> Low Stock
"Security Stock" -> Critical Stock

The French terms need to be reviewed.   

I could not figure out where STOCK.STATUS.MAXIMUM was used, so I did not change it.  

NOTE: To see these in action, I suggest using one of the production databases.

### CODE REVIEW

**Comment by jmcameron** [comment]
@mbayopanda Please review the French translations.  Thanks!

File: client/src/i18n/en/stock.json [hunk 1]
```diff
@@ -165,9 +165,9 @@
       "IN_STOCK"      : "In Stock",
       "LABEL"         : "Status",
       "MAXIMUM"       : "Maximum Stock Reached",
-      "MINIMUM"       : "Minimum Stock Reached",
+      "MINIMUM"       : "Low Stock",
       "OVER_MAX"      : "Over Maximum",
-      "SECURITY"      : "Security Stock Reached",
+      "SECURITY"      : "Critial Stock",
```
Comment (jniles) [inline client/src/i18n/en/stock.json]: Looks like a typo here - should be critical, not critial.

File: client/src/i18n/fr/stock.json [hunk 2]
```diff
@@ -169,10 +169,10 @@
       "IN_STOCK"      : "Disponible",
       "LABEL"         : "État",
       "MAXIMUM"       : "Stock Max Atteint",
-      "MINIMUM"       : "Stock Min Atteint",
+      "MINIMUM"       : "Stock faible",
       "OVER_MAX"      : "Stock Max Depassé",
-      "SECURITY"      : "Stock de Securité Atteint",
-      "STOCK_OUT"      : "Rupture de stock",
+      "SECURITY"      : "Stock Critique",
```
Comment (mbayopanda) [inline client/src/i18n/fr/stock.json:174]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

bors r+

**Pull request merged**